src/lib/Behat/PageElement/Fields/File.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement\Fields;
```

Comment:
Please add `declare(strict_types=1);`. Tests should follow the same coding standards as the source code. I suspect this is an issue among all tests related files but maybe you can align it slowly in future PRs.